docs/content/staking/faq.mdx

```diff
@@ -23,14 +31,14 @@ No, rewards are calculated the same for every node type.
 ### Can a validator change its fees?
 
 For the beta period we’re enforcing a delegation fee of 8% which cannot be changed.
+Any changes that nodes claim they have are not enforced by the protocol. 
```

Comment:
I'm confused by what this means

Reply:
Some nodes claim that they take a smaller cut, but in reality they just send a separate transaction to pay back their users for the difference. I added some clarification